tests/generate_large_data/helper_simulate_data.py

```
@@ -208,82 +286,250 @@ def generate_admissions_batch(patients_info, mean_records_per_pid):
+        include_labs: Whether to include lab test data
+        data_type: Type of data to generate ("random", "correlated", or "both")
+    """
+    os.makedirs(write_dir, exist_ok=True)
+    batch_size_patients = min(200_000, n_patients)
 
-    return concepts_data
+    # Initialize parquet writers
+    writers = {}
+    schema_written = False
+
+    for i in tqdm(range(n_patients // batch_size_patients)):
```

Ruff (0.8.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)